node/src/command_helper.rs

```diff
@@ -22,7 +22,7 @@
 use crate::service::FullClient;
```

Comment:
Can we move to `benchmarking.rs` to match the node template please?

Reply:
👍

Reply:
as in rename the whole file